client/src/lib/supabase/middleware.ts

```diff
@@ -78,60 +58,55 @@ export async function updateSession(request: NextRequest) {
 
   //simple pathname variable
   const pathname = request.nextUrl.pathname;
-  console.log("USER REQUEST PATHNAME: "+ pathname);
   
   //Auth user
   if (user) {
     const userEmail = user?.email;
     console.log("USER " + userEmail + " HAS AUTHENTICATED");
```

Comment:
let's remove console logs in the middleware too after a feature is implmented

Reply:
On it